src/components/newuser/newUserActions.tsx

```diff
@@ -177,6 +182,89 @@ export default function BeEventAdminForm({
           )}
         />
 
+        <FormField
```

Comment:
this is the wrong place to put this... it should be under a new page /admin/org/[id]/settings/page.tsx

or something like that.

Reply:
we dont want to overload the user when creating their account in the initial state

Reply:
Nevermind... lets keep this